nibabel/gifti/gifti.py

```diff
@@ -427,14 +425,12 @@ def add_gifti_data_array(self, dataarr):
         """
         if isinstance(dataarr, GiftiDataArray):
             self.darrays.append(dataarr)
-            self.numDA += 1
```

Comment:
Yes, sorry to be confusing - I saw you didn't need the line, but I was wondering if there were tests to make sure the computation was working right.